pages/item-new/_.vue

```diff
@@ -0,0 +1,369 @@
+
+        return edmDataProvider;
+      },
+      notificationUrl() {
+        return `https://classic.europeana.eu/portal/${this.$i18n.locale}/record${this.identifier}.html?utm_source=new-website&utm_medium=button`;
+      },
+      redirectNotificationsEnabled() {
+        return Boolean(Number(process.env.ENABLE_LINKS_TO_CLASSIC));
+      }
+    },
+
+    mounted() {
+      if (process.browser) {
+        this.$gtm.push({
+          itemCountry: langMapValueForLocale(this.fields.edmCountry, 'en').values[0],
+          itemDataProvider: langMapValueForLocale(this.coreFields.edmDataProvider, 'en').values[0],
+          itemProvider: langMapValueForLocale(this.fields.edmProvider, 'en').values[0],
+          itemRights: langMapValueForLocale(this.fields.edmRights, 'en').values[0]
+        });
+      }
+
+      // TODO: remove or is this useful for the new swiper/action?
+      window.addEventListener('message', (msg) => {
+        if (msg.data.event === 'updateDownloadLink') {
+          this.useProxy = (this.media.some((item) => item.about === msg.data.id));
+          this.selectedMedia.about = msg.data.id;
+        }
+      });
+    },
+
+    methods: {
+      annotationsByMotivation(motivation) {
+        return this.annotations.filter(annotation => annotation.motivation === motivation);
+      },
+
+      selectMedia(about) {
+        this.selectedMedia = about;
+      },
+
+      getSimilarItems() {
+        const noSimilarItems = { results: [] };
+        if (this.error) return noSimilarItems;
+
+        const dataSimilarItems = {
+          dcSubject: this.getSimilarItemsData(this.coreFields.dcSubject),
+          dcType: this.getSimilarItemsData(this.title),
+          dcCreator: this.getSimilarItemsData(this.coreFields.dcCreator),
+          edmDataProvider: this.getSimilarItemsData(this.fields.edmDataProvider)
+        };
+
+        return search({
+          query: similarItemsQuery(this.identifier, dataSimilarItems),
+          rows: 4,
+          profile: 'minimal',
+          facet: ''
+        }, {
+          origin: this.$route.query.recordApi
+        })
+          .catch(() => {
+            return noSimilarItems;
+          });
+      },
+
+      getSimilarItemsData(value) {
+        if (!value) return;
+
+        const data = langMapValueForLocale(value, this.$i18n.locale).values;
+        if (!data) return;
+
+        return data.filter(item => typeof item === 'string');
+      }
+    },
+
+    head() {
+      return {
+        title: this.metaTitle,
+        meta: [
+          { hid: 'title', name: 'title', content: this.metaTitle },
+          { hid: 'description', name: 'description', content: this.metaDescription },
+          { hid: 'og:title', property: 'og:title', content: this.metaTitle },
+          { hid: 'og:description', property: 'og:description', content: this.metaDescription },
+          { hid: 'og:image', property: 'og:image', content: this.selectedMediaImage.src ? this.selectedMediaImage.src : '' },
```

Comment:
See my comment above about setting the selectedMedia for og metadata.

Until we allow users to load the page with a specific media resource selected at page load time(preserved in the URL so it is transparent for bots), this is useless I believe.